tools/autograd/derivatives.yaml

```diff
@@ -211,6 +236,7 @@
 
 - name: alias(Tensor(a) self) -> Tensor(a)
   self: grad
+  result: auto_element_wise
```

Comment:
I know you played the "but `non_differentiable` did it this way", but I just *hate* the way formulas like this scan. It looks so weird.

Reply:
Do you think we should start adding boolean flags in the same spirit as what we do in native_functions.yaml to this file? We currently don't and that's why I followed the current practice.
But maybe we should do that when we do the codegen refactor to move these to boolean flags (same thing for output_differentiability and non_differentiable)